Makefile

```diff
@@ -1,6 +1,9 @@
 install:
 	obt install --verbose
 
+verify:
+	nbt verify --skip-layout-checks
+
 test:
 	obt build
-	obt verify
+	nbt verify --skip-layout-checks
```

Comment:
instead of duplicating verify, can call the `verify` task from `test` (and maybe have another task called `build` that it also calls), i.e. `test: build verify`